src/Functional.CQS.AOP.IoC.SimpleInjector/Functional.CQS.AOP.IoC.SimpleInjector.csproj

```diff
@@ -0,0 +1,38 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Description>For use with SimpleInjector dependency injection container.  Provides extension methods on Container that make registering Functional.CQS handler implementations easy.
+- Functional.CQS.IQueryHandler&lt;TQuery, TResult&gt;
+- Functional.CQS.IAsyncQueryHandler&lt;TQuery, TResult&gt;
+- Functional.CQS.ICommandHandler&lt;TCommand, TError&gt;
+- Functional.CQS.IAsyncCommandHandler&lt;TCommand, TError&gt;</Description>
+    <RepositoryUrl></RepositoryUrl>
+    <Copyright>2019</Copyright>
+    <PackageLicenseUrl>https://github.com/RyanMarcotte/Functional.CQS/blob/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/RyanMarcotte/Functional.CQS</PackageProjectUrl>
+    <PackageTags>functional CQS netstandard netcore simpleinjector aop ioc</PackageTags>
+    <Authors>Ryan Marcotte</Authors>
+    <Company>Ryan Marcotte</Company>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Projects\Functional.CQS\src\Functional.CQS.AOP.IoC.SimpleInjector\Functional.CQS.AOP.IoC.SimpleInjector.xml</DocumentationFile>
```

Comment:
For both the `Debug` and `Release` configurations, can the machine-specific `DocumentationFile` paths be changed to relative paths?
-----------
This comment also goes for `Functional.CQS.AOP.CommonTestInfrastructure.csproj`.